modules/olm-installing-operators-from-operatorhub-configure.adoc

```diff
@@ -0,0 +1,87 @@
+// Module included in the following assemblies:
+//
+// * operators/user/olm-installing-operators-in-namespace.adoc
+// * operators/admin/olm-adding-operators-to-cluster.adoc
+// * post_installation_configuration/preparing-for-users.adoc
+//
+// Module watched for changes by Ecosystem Catalog team:
+// https://projects.engineering.redhat.com/projects/RHEC/summary
+
+
+[id="olm-installing-operators-from-operatorhub-configure_{context}"]
+= Configuring {product-title} to use Red Hat Operators
+
+In {product-title}, Red Hat Operators are not available by default. You can access and install these Operators if you have a pull secret from the Red Hat OpenShift Cluster Manager site by editing the Operator Hub custom resource (CR). 
+
+.Prerequisites
+
+* Ensure that you have downloaded the link:https://cloud.redhat.com/openshift/install/pull-secret[pull secret from the Red Hat OpenShift Cluster Manager site] as shown in the installation procedure.
+
+.Procedure
+
+To access the Red Hat Operators in a {product-title} cluster: 
+
+. Edit the Operator Hub CR using the web console or CLI:
+
+.. Using the CLI:
+
+... Edit the `OperatorHub` CR object:
++
+[source,terminal]
+----
+$ oc edit OperatorHub cluster
```

Comment:
s/$ oc edit OperatorHub cluster/$ oc edit operatorhub cluster ? (Did you test to see if it works with caps? If so, ignore.)

Reply:
@bobfuru Both versions work. The object YAML uses `OperatorHub`, so I went with that.

```
apiVersion: config.openshift.io/v1
kind: OperatorHub

```